ipatests/prci_definitions/temp_commit.yaml

```diff
@@ -68,7 +68,7 @@ jobs:
       class: RunPytest
       args:
         build_url: '{fedora-latest/build_url}'
-        test_suite: test_integration/test_REPLACEME.py
```

Comment:
Instead of replacing test_REPLACEME, please remplace the whole temp commit block by:
```
  fedora-latest/test_adtrust_install:
    requires: [fedora-latest/build]
    priority: 100
    job:
      class: RunPytest
      args:
        build_url: '{fedora-latest/build_url}'
        test_suite: test_integration/test_adtrust_install.py
        template: *ci-master-latest
        timeout: 3600
        topology: *master_1repl
```
The way the temp commit template is presented makes it unclear that there are other parameters to change (e.g. timeout). When you want to run a test, please find the complete entry in gating or in the nightlies and replace the temp commit block by it.